cypress-tests/cypress/e2e/PaymentTest/00004-NoThreeDSAutoCapture.cy.js

```diff
@@ -25,46 +25,43 @@ describe("Card - NoThreeDS payment flow test", () => {
     });
 
     it("create-payment-call-test", () => {
-      let data = getConnectorDetails(globalState.get("connectorId"))["card_pm"][
-        "PaymentIntent"
-      ];
-      let req_data = data["Request"];
-      let res_data = data["Response"];
+      const data = getConnectorDetails(globalState.get("connectorId"))[
+        "card_pm"
+      ]["PaymentIntent"];
+
       cy.createPaymentIntentTest(
         fixtures.createPaymentBody,
-        req_data,
-        res_data,
+        data,
         "no_three_ds",
         "automatic",
         globalState
       );
+
       if (should_continue)
-        should_continue = utils.should_continue_further(res_data);
+        should_continue = utils.should_continue_further(data);
     });
 
     it("payment_methods-call-test", () => {
       cy.paymentMethodsCallTest(globalState);
     });
 
     it("Confirm No 3DS", () => {
-      let data = getConnectorDetails(globalState.get("connectorId"))["card_pm"][
-        "No3DSAutoCapture"
-      ];
-      let req_data = data["Request"];
-      let res_data = data["Response"];
-      cy.confirmCallTest(
-        fixtures.confirmBody,
-        req_data,
-        res_data,
-        true,
-        globalState
-      );
+      const data = getConnectorDetails(globalState.get("connectorId"))[
+        "card_pm"
+      ]["No3DSAutoCapture"];
+
+      cy.confirmCallTest(fixtures.confirmBody, data, true, globalState);
+
       if (should_continue)
-        should_continue = utils.should_continue_further(res_data);
+        should_continue = utils.should_continue_further(data);
     });
 
     it("retrieve-payment-call-test", () => {
-      cy.retrievePaymentCallTest(globalState);
+      const data = getConnectorDetails(globalState.get("connectorId"))[
```

Comment:
glad you asked. data exist only to fetch the `Configs` and nothing else. in the `support/commands.js` (image attached below), we're extracting `Configs` from `data`. this is done specifically keeping `CONNECTOR_CREDENTIAL` in mind so that we can pass `mca_id` directly which is used in verification (assertion check). 

<img width="583" alt="image" src="https://github.com/user-attachments/assets/68c85007-b61a-47de-acef-959e8d127970">

the entire field can be removed if we stop doing `mca_id` check.

Reply:
Are we not verifying the payment status ?

Reply:
yes. remove this mca_id check?